libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -92,5 +92,14 @@ public QnAMakerOptions()
         /// <seealso cref="RankerTypes"/>
         [JsonProperty("rankerType")]
         public string RankerType { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether  enable Precise Answer.
+        /// </summary>
+        /// <value>
+        /// To enable Precise Answer or not.
```

Comment:
Copyedit, improvement to doc quality needed.

Reply:
Fixed.

---
In reply to: [430553619](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430553619) [](ancestors = 430553619)